codec-stomp/src/main/java/io/netty/handler/codec/stomp/StompSubframeEncoder.java

```diff
@@ -67,7 +68,10 @@ private static ByteBuf encodeFrame(StompHeadersSubframe frame, ChannelHandlerCon
         buf.writeBytes(frame.command().toString().getBytes(CharsetUtil.US_ASCII));
         buf.writeByte(StompConstants.LF);
         try {
-            frame.headers().forEachEntry(new AsciiHeadersEncoder(buf, SeparatorType.COLON, NewlineType.LF));
+            AsciiHeadersEncoder headersEncoder = new AsciiHeadersEncoder(buf, SeparatorType.COLON, NewlineType.LF);
+            for (Entry<CharSequence, CharSequence> entry : frame.headers()) {
+                headersEncoder.encode(entry);
+            }
         } catch (Exception ex) {
```

Comment:
I don't think this is needed anymore. The exception can be removed from `AsciiHeadersEncoder.encode` too.